.nvmrc

```
@@ -1 +1 @@
-v8.15.0
+v12.16.3
```

Let's use the v12 LTS instead of a fixed version here.

```suggestion
lts/erbium
```

---

nice! I was looking for this, but couldn't find it. Where did you got this from?